test/realizations/catchments/Bmi_C_Pet_IT.cpp

```diff
@@ -140,7 +132,6 @@ void Bmi_C_Pet_IT::SetUp() {
                          "            \"bmi_c\": {"
                          "                \"model_type_name\": \"" + model_type_name[i] + "\","
                          "                \"library_file\": \"" + lib_file[i] + "\","
-                         "                \"forcing_file\": \"" + forcing_file[i] + "\","
```

Comment:
The corresponding correct setting appears in the `"forcing": { "path": forcing_file[i] }` block just below.